config.ml

```diff
@@ -1,4 +1,4 @@
-(* mirage >= 4.5.1 & < 4.6.0 *)
+(* mirage >= 4.6.0 & < 4.6.1 *)
```

Comment:
can we do < 4.7.0, and make mirage 4.x.y compatible for every y. I used 4.6.0 to ensure that.

I guess that should be documented properly... :)